recipes/perl-vcftools-vcf/0.840/run_test.pl

```diff
@@ -0,0 +1 @@
+use Vcf;
```

Comment:
I noticed this has been happening lately too.

I changed my tests to -

test:
  commands:
    - perl -e "use SomeLib"

But its not awesome because then it doesn't check for versioning.